sequencing/ATACSeq/preprocessing/alignment.sh

```diff
@@ -64,9 +69,9 @@ then
 	# remove reads with q < 30, unmapped, mate unmapped, secondary alignment, reads failing platform
 	# only keep properly paired reads
 	echo "filtering aligned reads"
-    samtools view -F 524 -f 2  -q 30 -u ${ALIGNEDDIR}/${sampleName}_noMT.bam | samtools sort -n /dev/stdin -o ${ALIGNEDDIR}/${sampleName}_q30.tmp.nmsrt.bam
-    samtools view -h ${ALIGNEDDIR}/${sampleName}_q30.tmp.nmsrt.bam | $(which assign_multimappers.py) -k $multimap --paired-end | samtools fixmate -r /dev/stdin ${ALIGNEDDIR}/${sampleName}_q30.tmp.nmsrt.fixmate.bam
-	
+  samtools view -F 524 -f 2  -q 30 -u ${ALIGNEDDIR}/${sampleName}_noMT.bam | samtools sort -n /dev/stdin -o ${ALIGNEDDIR}/${sampleName}_q30.tmp.nmsrt.bam
+  samtools view -h ${ALIGNEDDIR}/${sampleName}_q30.tmp.nmsrt.bam | $(which assign_multimappers.py) -k $multimap --paired-end | samtools fixmate -r /dev/stdin ${ALIGNEDDIR}/${sampleName}_q30.tmp.nmsrt.fixmate.bam
```

Comment:
I never liked this line in the original script. Does this actually work? It requires `assign_multimappers.py` to be on $PATH. For me at least, this is not the case. I feel like it would be better to just put the path to this python file into the config file:
```
MULTIMAPPER_PY_PATH="path/to/assign_multimappers.py"
```